public/app/features/library-panels/components/PanelLibraryOptionsGroup/PanelLibraryOptionsGroup.tsx

```diff
@@ -92,3 +111,7 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
     `,
   };
 });
+
+const mapDispatchToProps: MapDispatchToProps<DispatchProps, Props> = { changePanelPlugin };
+
+export const PanelLibraryOptionsGroup = connect(undefined, mapDispatchToProps)(PanelLibraryOptionsGroupUnconnected);
```

Comment:
I think you should use `useDispatch` instead as this is an FC.

Reply:
Ah, was trying to do things like `VisualizationTab`, but I guess that makes sense, will change.